### PR TITLE
[IOS-554]: Add pull to refresh on Browser

### PR DESCRIPTION
**Pull request by yuliaveres**
According to design we need to have a way to reload current page on Browser controller. As it was agreed - we are going to add Pull to refresh on webView's scrollView

### CODE REVIEW

File: Toshi/SOFAWebview/SOFAWebController.swift [hunk 1]
```diff
@@ -585,6 +597,10 @@ extension SOFAWebController: WKUIDelegate {
 
         return nil
     }
+
+    public func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+        refreshControl.endRefreshing()
```
Comment (designatednerd) [inline Toshi/SOFAWebview/SOFAWebController.swift]: does `didFinish` also get called when there's an error?

Comment (yuliaveres) [reply Toshi/SOFAWebview/SOFAWebController.swift]: not sure, but should we end refresh in case it fails ?

Comment (yuliaveres) [reply Toshi/SOFAWebview/SOFAWebController.swift]: i will implement didFinish as well

Comment (designatednerd) [reply Toshi/SOFAWebview/SOFAWebController.swift]: I would not recommend auto-refreshing if it fails - that'll infinite loop if there's a server error, or the user is not connected to the internet. 

and i think you need to implement the error one, this method is didFinish.

Comment (yuliaveres) [reply Toshi/SOFAWebview/SOFAWebController.swift]: @designatednerd done

**Pull request merged**